### PR TITLE
refactor(material-experimental/mdc-progress-bar): remove wrapper element for progress bar

### PR DESCRIPTION
**Pull request by annieyw**
refactor progress bar to have same structure as progress spinner (without wrapper element)

related: https://github.com/angular/components/pull/22429

### CODE REVIEW

**Comment by annieyw** [comment]
Seems like the issue with the tests is that the original progress bar has a wrapper div around the different bars but as discussed in #22429 progress bar and progress spinner should have the same a11y setup. Any ideas on which we should prioritize? (align with original progress bar or have mdc progress bar/spinner have same a11y)

File: src/material-experimental/mdc-progress-bar/progress-bar.scss [hunk 1]
```diff
@@ -1,22 +1,20 @@
...
-@include mdc-linear-progress.core-styles($query:
-  mdc-helpers.$mat-base-styles-without-animation-query);
+@include mdc-linear-progress.core-styles($query: mdc-helpers.$mat-base-styles-query);
 
 .mat-mdc-progress-bar {
   // Explicitly set to `block` since the browser defaults custom elements to `inline`.
   display: block;
 
   &._mat-animation-noopable {
+    // Disabled the loading animations.
```
Comment (crisbeto) [inline src/material-experimental/mdc-progress-bar/progress-bar.scss:11]: nit: disabled -> disable

**Review by crisbeto — APPROVED** [review]
The changes LGTM. I think that we should keep the DOM as close to MDC as possible and update the tests to expect more than one root node inside the progress bar. I don't think that screen readers care whether it's one root node with `aria-hidden` or multiple siblings with `aria-hidden`.

**Comment by annieyw** [comment]
Animations were still running with `NoopAnimationsModule` with the previous fix. Added some more specific selectors to disable animations (like in progress spinner)

**Pull request merged**